website/static/js/institutionProjectSettings.js

```diff
@@ -111,9 +195,39 @@ var ViewModel = function(data) {
     };
 };
 
+/**
+ * get node tree for treebeard from API V1
+ */
+ViewModel.prototype.fetchNodeTree = function(treebeardUrl) {
+        var nodesOriginal = {};
+        var self = this;
+        return $.ajax({
+            url: treebeardUrl,
+            type: 'GET',
+            dataType: 'json'
+        }).done(function (response) {
+            nodesOriginal = projectSettingsTreebeardBase.getNodesOriginal(response[0], nodesOriginal);
+            self.nodeParent = response[0].node.id;
+            self.hasChildren(Object.keys(self.nodesOriginal).length > 1);
```

Comment:
I am seeing this modal even for projects that don't have components.

![osf___osf4i_test_settings](https://cloud.githubusercontent.com/assets/2379650/15549733/008067c8-227c-11e6-83eb-1fe5f7f992f8.png)

The problem is this line; you are getting the keys for `self.nodesOriginal`, which is an `observable` that happens to have 5 methods, so this will always be `true`. I think this should be changed to `nodesOriginal`.